transactions/types.go

```diff
@@ -55,9 +64,17 @@ type SendTxArgs struct {
 	Input types.HexBytes `json:"input"`
 	Data  types.HexBytes `json:"data"`
 
-	// additional data
+	// additional data - version SendTxArgsVersion1
```

Comment:
hmm not sure why we decided to just copy all the fields from `ethapi.TransactionArgs` (raw parameters needed to send a tx to the chain), I kind of like to keep that as a separate struct and then have all the remaining "metadata" as additional fields. Then the lower level modules that are not interested in the metadata can just consume the original type.

Reply:
@dlipicar this is the first step of improvements and simplifications I want to do, we can discuss that for some of the next steps for sure, but this was the easiest now since any bigger change will require updating `Transactor` that I don't want to touch yet, since I didn't want to break any endpoint that we already have and is being used by the mobile app.